gax-java/gax/src/main/java/com/google/api/gax/rpc/ClientContext.java

```diff
@@ -189,6 +193,9 @@ public static ClientContext create(StubSettings settings) throws IOException {
       endpointContext = endpointContext.withGDCH();
       // Resolve the new endpoint with the GDC-H flow
       endpoint = endpointContext.resolvedEndpoint();
+      // Skip S2A in GDC-H.
+      mtlsEndpoint = "";
+      endpointOverride = "";
```

Comment:
@xmenxk , can you confirm we don't want to try and use mTLS via S2A in GDC-H flow?

Reply:
Yes I think we can skip S2A for this.